zebra/zebra_vxlan.h

```diff
@@ -26,6 +26,16 @@
 extern "C" {
 #endif
 
+DECLARE_MGROUP(ZEBRA);
+
+DECLARE_MTYPE(VLAN_CHANGE_ARR);
+
+#define ZEBRA_DPLANE_BR_STATE_DISABLED	 0x01
```

Comment:
I'd given an example of some similar "neutral" values used in the dplane that are in the dplane header. why not put these there too?

Reply:
Will do